roles/openshift_master/tasks/main.yml

```diff
@@ -172,14 +172,9 @@
   register: start_result
   notify: Verify API Server
 
-- name: Check for non-HA master service presence
-  command: systemctl show {{ openshift.common.service_type }}-master.service
-  register: master_svc_show
-  changed_when: false
-
-- name: Stop and disable non-HA master when running HA
+- name: Stop and disable non HA master when running HA
   service: name={{ openshift.common.service_type }}-master enabled=no state=stopped
-  when: openshift_master_ha | bool and 'LoadState=not-found' not in master_svc_show.stdout
```

Comment:
Yikes, not sure how this happened but this is explicitly undo'ing the fix that went in with https://github.com/openshift/openshift-ansible/pull/1971/files. I botched something with submitting it separately and then syncing the branches. Removing this from this PR.